docs/Using-Fleet/Adding-hosts.md

```diff
@@ -3,14 +3,10 @@
 - [Signing installers](#signing-installers)
 - [Plain osquery](#plain-osquery)
 
-Fleet is powered by the open source osquery tool. To add a host to Fleet, you must install osquery on this host.
-
 The recommended way to install osquery and add your host to Fleet is with an osquery installer. Fleet provides the tools to generate an osquery installer with the `fleetctl package` command.
```

Comment:
```suggestion
The recommended way to add your host to Fleet is with an osquery installer. Fleet provides the tools to generate an osquery installer with the `fleetctl package` command.
```

Reply:
What do you think about removing "install osquery and" from the first sentence? 

This way, the first sentence the user reads is focused on only adding your hosts to Fleet.